alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -177,8 +188,11 @@ private function getAutoEmbedCode($targetId = null)
 			$config[$key] = json_decode($val);
 		}
 
+
 		$config["partnerId"] = $this->partnerId;		
 		$config["uiConfId"] = $this->uiconfId;
+		$config["uiConf"] = $this->playerConfig;
```

Comment:
playerConfig is missing in the class params definition.

Reply:
@tanyaLibatter there's no uiConf in the player code, you need to pass the config as is.
The config option would be same format as player expects it to be, e.g. 
```json
{
   "playback": {...},
   "plugins": {
      "pluginA": {...},
      "pluginB": {...}
   }
}
```

Reply:
Also, need to handle situation where config is passed from client side and the we need to merge accordingly.